Fabric.Identity.API/Views/Account/Login.cshtml

```diff
@@ -84,24 +80,5 @@
                 </div>
             </div>
             <br><br><br>
-            <div class="footerWrapper" align="center">
-                <div class="footerBottom" align="center">
```

Comment:
Not that we need it, but what if we leave these divs and only remove the links below?

Reply:
Ya I'll put them back for a possible future use.